### DISCUSSION TITLE
centos环境 ，上载速度只有500k/s

### DISCUSSION THREAD

**Discussion by learnerph2025** [houtianze/bypy#641] [Q&A]
百度vip会员， centos下用百度网盘客户端 上载可以达到5-8M/s。但是用bypy上载速度只有500k/s
下载速度没有问题，可以到5-9M/s
请问可能是哪儿配置不对么？

**Comment by wangzyphysics** [comment]
解决了吗...

**Reply by Dalaoyel** [reply]
我也遇到了，超级vip也不管用

**Comment by Lssg97** [comment]
> 百度vip会员， centos下用百度网盘客户端 上载可以达到5-8M/s。但是用bypy上载速度只有500k/s 下载速度没有问题，可以到5-9M/s 请问可能是哪儿配置不对么？

Oracle 的 VPS，Ubuntu 22.04，上传也是限制在 475Kb/s，这应该不是个例awa

**Comment by Dalaoyel** [comment]
尝试很多次，都是500kb/s。他们说是作者的中转服务器有限制。
用linux阿里云吧，可以达到30mb/s的上传速度。希望对你有用


所以呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "houtianze/bypy"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月12日(星期五) 晚上11:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [houtianze/bypy] centos环境 ，上载速度只有500k/s (Discussion #641)





  
百度vip会员， centos下用百度网盘客户端 上载可以达到5-8M/s。但是用bypy上载速度只有500k/s 下载速度没有问题，可以到5-9M/s 请问可能是哪儿配置不对么？
  
Oracle 的 VPS，Ubuntu 22.04，上传也是限制在 475Kb/s，这应该不是个例awa

**Comment by houtianze** [comment]
这个速度与中转服务器没有关系的，中转服务器只是授权转发，不涉及任何文件传输和其他pcs api ，而且最新版已经完全不用中转服务器，都是直连百度服务器。这个速度几乎肯定是百度那边设定的。On 14 May 2023, at 9:57 AM, 达劳耶尔 ***@***.***> wrote:﻿
尝试很多次，都是500kb/s。他们说是作者的中转服务器有限制。

用linux阿里云吧，可以达到30mb/s的上传速度。希望对你有用





所以呢

***@***.***







&nbsp;









------------------&nbsp;原始邮件&nbsp;------------------

发件人:                                                                                                                        "houtianze/bypy"                                                                                    ***@***.***&gt;;

发送时间:&nbsp;2023年5月12日(星期五) 晚上11:36

***@***.***&gt;;

***@***.******@***.***&gt;;

主题:&nbsp;Re: [houtianze/bypy] centos环境 ，上载速度只有500k/s (Discussion #641)











  

百度vip会员， centos下用百度网盘客户端 上载可以达到5-8M/s。但是用bypy上载速度只有500k/s 下载速度没有问题，可以到5-9M/s 请问可能是哪儿配置不对么？

  

Oracle 的 VPS，Ubuntu 22.04，上传也是限制在 475Kb/s，这应该不是个例awa

**Comment by Dalaoyel** [comment]
win10上的上传速度不受vip限制。你的意思是说百度限制了linux的上传速度吗，而且开vip还解决不了。这显然是不合理的

所以呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "houtianze/bypy"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月14日(星期天) 上午10:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [houtianze/bypy] centos环境 ，上载速度只有500k/s (Discussion #641)





 这个速度与中转服务器没有关系的，中转服务器只是授权转发，不涉及任何文件传输和其他pcs api ，而且最新版已经完全不用中转服务器，都是直连百度服务器。这个速度几乎肯定是百度那边设定的。On 14 May 2023, at 9:57 AM, 达劳耶尔 ***@***.***&gt; wrote:﻿
 尝试很多次，都是500kb/s。他们说是作者的中转服务器有限制。
 
 用linux阿里云吧，可以达到30mb/s的上传速度。希望对你有用
 
 
 
 
 
 所以呢
 
 ***@***.***
 
 
 
 
 
 
 
 &amp;nbsp;
 
 
 
 
 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------
 
 发件人:                                                                                                                        "houtianze/bypy"                                                                                    ***@***.***&amp;gt;;
 
 发送时间:&amp;nbsp;2023年5月12日(星期五) 晚上11:36
 
 ***@***.***&amp;gt;;
 
 ***@***.******@***.***&amp;gt;;
 
 主题:&amp;nbsp;Re: [houtianze/bypy] centos环境 ，上载速度只有500k/s (Discussion #641)
 
 
 
 
 
 
 
 
 
 
 
   
 
 百度vip会员， centos下用百度网盘客户端 上载可以达到5-8M/s。但是用bypy上载速度只有500k/s 下载速度没有问题，可以到5-9M/s 请问可能是哪儿配置不对么？
 
   
 
 Oracle 的 VPS，Ubuntu 22.04，上传也是限制在 475Kb/s，这应该不是个例awa

**Comment by Lssg97** [comment]
> 这个速度与中转服务器没有关系的，中转服务器只是授权转发，不涉及任何文件传输和其他pcs api ，而且最新版已经完全不用中转服务器，都是直连百度服务器。这个速度几乎肯定是百度那边设定的。

新尝试：加上“--processes”参数，可能是一个加快上传速度的解决方法。
我最近在用 `bypy -v --processes 3 ./ /`，三条线程每个都被限制在 475Kb/s，但是加在一起总速就快了。不过没有试过设置更多的线程数。

**Reply by DuoAp** [reply]
貌似一个文件还是只有一个线程上传，如果是文件数量少但单个文件大的情况还是没有效果。

**Reply by houtianze** [reply]
是的，一个文件是单线程上传/下载的，只有多个文件才会多进程上传/下载。

**Comment by yinworth** [comment]
我也遇到了一样的问题，win10系统，svip用户，并且使用了自己注册的Appkey和Secretkey授权认证，上传速度只能在475kb/s

**Comment by MingsYang** [comment]
你们下载速度都能有5-8M/s吗，为啥我这边显示速度只有几百k
![image](https://github.com/houtianze/bypy/assets/15760556/a9c48d56-84d8-4c74-8a9a-718f7f3ba7f8)

**Reply by MingsYang** [reply]
![image](https://github.com/houtianze/bypy/assets/15760556/c527438b-f1de-48dc-9308-1b6d11ffd341)
这是开了会员的速度，也太慢了吧

**Reply by Dalaoyel** [reply]
开了会员下载能跑满，上传跑不满

**Comment by iamwangyabin** [comment]
上传肯定是百度那边限制了，下载很快

**Reply by fallinwind** [reply]
确实，怎么试都是479kb/s，肯定是百度限制了linux的上传速率

**Comment by grapebabe** [comment]
作者你好，我是乌班图环境。
有几个提问麻烦帮忙解答一下。
1.上传速率只有400-500k，如何提高上传速率？我已经开始开了百度超级会员
2.如果上传指定路径文件？
非常感谢
祝你一切顺利

**Reply by fallinwind** [reply]
上传好像是被百度那边限制了，试了下可以每个文件都放到后台同时上传，虽然每个文件只有四五百k，也没找到更好的方法。

**Comment by saibeiyungao** [comment]
bypy ... --processes 10

<W> [15:57:28] Running multiple (10) processes for Sync up, I don't know if things will go awry...
......

**Reply by saibeiyungao** [reply]
![updata](https://github.com/user-attachments/assets/d72845e2-88fd-4d04-bea4-f221f6ac08e5)
4-8M

**Comment by houtianze** [comment]
程序不是多线程的，所以单个文件没啥办法。多个文件的话可以加个`--processes`参数并行（文件）上传，但是可能百度那边会返回更多的错误。

**Comment by changshengEVA** [comment]
单文件似乎可以试试分卷压缩再多线程下载

**Comment by marsxw** [comment]
<img width="1083" height="401" alt="image" src="https://github.com/user-attachments/assets/d0b3b238-4c88-483e-8430-7afee40087bb" />
svip 只有几百K速度，不知道会员的意义在哪里

**Reply by houtianze** [reply]
根据之前的反馈，应该是pcs api并没和vip会员连起来，如果多文件的话可以试试这个`--processes`命令行开关多*进程*上传/下载，能提高一点整体速度，单个文件的传输速度恐怕是没啥办法提高了。

**Comment by ruinmi** [comment]
真的是死死卡在500kb啊，真的服了
